_includes/cloud-db/cloud-summary-db-version.md

```diff
@@ -0,0 +1 @@
+Each database is displayed with a version - this represents the version of Featurebase that the database is running. 
```

Comment:
A database version represents the FeatureBase version the database [belongs to | runs on].

(pick either belongs to/runs on) :)